tuple-table/src/test/java/com/iluwatar/tuppletable/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package com.iluwatar.tuppletable;
+
+import org.junit.Test;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class AppTest {
+    /**
+     *Test for testing the connection ensuring the code works as per design
+     */
+    @Test
+    public void shouldExecuteWithoutException() {
+        assertDoesNotThrow(() -> App.main(new String[]{}));
+    }
+
+    /**
+     *
+     * Test for public methods for MemberTupleDAO
+     * @throws SQLException
+     */
+
+    @Test
+    public void testForPublicMethods() throws SQLException {
```

Comment:
This kind of test name is unacceptable. Use short and focused tests.

Reply:
gave proper name, sorry for the dirty naming.